plugins/web-plugin/src/test/java/setting/JettySettings.java

```diff
@@ -18,41 +18,41 @@
 public class JettySettings {
 
     private static final Logger LOG = LoggerFactory.getLogger(JettySettings.class);
-    
+
     public static final String WAR_PATH = "src/test/resources/test-web-app.war";
     public static final int PORT = 8181;
-    
+
     private static boolean dunit = true;
-    
+
     private Server server;
-    
+
     @Before(order = 999)
     public void startJetty() throws Exception {
-    
+
         if (dunit) {
-            // stop jetty after all tests
+            // stop jetty after all features
             getRuntime().addShutdownHook(new Thread(new Runnable() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)